### DISCUSSION TITLE
Custom description and metadata fields for dbt Cloud assets

### DISCUSSION THREAD

**Discussion by mudravrik** [dagster-io/dagster#24991] [Q&A]
Hi all!

Is there any way to add custom metadata/description to assets imported by [load_assets_from_dbt_cloud_job()](https://docs.dagster.io/_apidocs/libraries/dagster-dbt#assets-dbt-cloud)? I see the arguments in docstring, but there is no actual argument for it in function signature.
There are also bunch of "hints" in the source code, that at least metadata should be modify-able by user, but I can't see how actually it may be done.

**Comment by mlarose** [comment]
Hi @mudravrik,

You are right, the docstring for this function is misleading.

At this time, it's not possible to add custom metadata with the current implementation. You could roll your own version of the function, using a custom derived class of `DbtCloudCacheableAssetsDefinition` overloading `_build_dbt_cloud_assets_cacheable_data` which would allow `DagsterDbtTranslator.get_metadata` to be customized. 

However, the folks working on integration here have a slightly different approach on the roadmap decoupling the translator from the `DbtCloudCacheableAssetsDefinition`. So the above suggestion is possibly not the future proof way, but could be a temporary fix to your need.
